ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -42,6 +43,7 @@ component:
   heater: "Component which provides heat to media."
   humidifier: "Component which humidifies."
   lamp: "The component of a lighting device that emits the light."
+  motor: "The component of a compressor."
```

Comment:
Motor: a machine, especially one powered by electricity or internal combustion, that supplies motive power for a vehicle or for some other device with moving parts.

Reply:
the description has been changed